jira-doc.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import print_function
+from collections import Counter
+from jira import JIRA
+from datetime import datetime, timedelta
+import json
+import ast
+import re
+
+def get_list(sprint):
+    query_string = ('project=JD and status="Done" and Sprint="' + sprint +'"')
+    old_sprint = (int(sprint)-1)
```

Comment:
this may be a sprint from another project :( You need to find a more reliable way of determining the previous sprint in JD